src/Skybrud.Colors/ColorUtils.cs

```diff
@@ -1430,6 +1430,84 @@ public static bool TryParse(string str, out IColor color)
 
         #endregion
 
+        #region Groups of Colors
+
+        /// <summary>
+        /// Returns a list of IColors which represent even gradient steps between two colors.
+        /// </summary>
+        /// <param name="StartColor"></param>
+        /// <param name="EndColor"></param>
+        /// <param name="NumberofIntervals">Quantity of IColors to return</param>
+        /// <returns>NOTE: The first color returned is identical to the StartColor,
+        /// but the last color might not be identical to the EndColor, since the last color is calculated
+        /// and rounding in the math used will affect the outcome slightly. </returns>
+        public static IEnumerable<IColor> GetGradientColors(IColor StartColor, IColor EndColor, int NumberofIntervals)
```

Comment:
.NET convention is to use lower camel casing for parameter names. Could you change the parameter names to reflect this?